custom_components/localtuya/cover.py

```diff
@@ -97,6 +118,15 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
                 TuyaCache(device, config_entry.data[CONF_FRIENDLY_NAME]),
                 config_entry,
                 device_config[CONF_ID],
```

Comment:
Remove unnecessary old code.

Reply:
I deleted:
```
TuyaCache(device, config_entry.data[CONF_FRIENDLY_NAME]),
                config_entry,
                device_config[CONF_ID],
```

So that it now reads:
```
    for device_config in entities_to_setup:
        dps[device_config[CONF_ID]] = None
        covers.append(
            LocaltuyaCover(
            )
        )
```

Is that correct?

Reply:
The original code was correct, I was referring to the commented code that was used before.

Reply:
Gotcha, I thought my edit seemed wrong. Thanks for the clarification.